Update README.md (#4)
thepolicylab-projectportals/.github: README.md

# .github

> 
> 

Reviewer: ```suggestion

This repository is for GitHub Actions workflows shared amongst all the project repositories in the organization.

## pa11y-ci workflow
```

> This repository is a standalone GitHub Action that automates the process of running Pa11y accessibility tests on other Project Portal repositories. Pa11y helps ensure that your web pages are accessible to everyone, including those with disabilities. 

Reviewer: ```suggestion
This workflow automates the process of running Pa11y accessibility tests on other Project Portal repositories. Pa11y helps ensure that your web pages are accessible to everyone, including those with disabilities. 
```
